File: Rio/Rio.Web/Modules/ServerTypes/Workspace/EPaperSize.ts
import { Decorators } from "@serenity-is/corelib";

export enum EPaperSize {
    A4 = 4,
    A5 = 5
}
Decorators.registerEnumType(EPaperSize, 'Rio.Workspace.EPaperSize', 'Workspace.EPaperSize');

File: Rio/Rio.Web/enums/EPaperSize.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace Rio.Workspace.enums
{
    [EnumKey("Workspace.EPaperSize")]
    public enum EPaperSize
    {
        [Description("A4")]
        A4 = 4,
        [Description("A5")]
        A5 = 5
    }
}